apps/storybook-react-native/.storybook/FontLoader.js

```diff
@@ -8,12 +8,12 @@ const FontLoader = ({ children }) => {
   useEffect(() => {
     (async () => {
       await Font.loadAsync({
-        'EuclidCircularB-Regular': require('../assets/fonts/EuclidCircularB-Regular.otf'),
-        'EuclidCircularB-Bold': require('../assets/fonts/EuclidCircularB-Bold.otf'),
-        'EuclidCircularB-RegularItalic': require('../assets/fonts/EuclidCircularB-RegularItalic.otf'),
-        'EuclidCircularB-BoldItalic': require('../assets/fonts/EuclidCircularB-BoldItalic.otf'),
-        'EuclidCircularB-Medium': require('../assets/fonts/EuclidCircularB-Medium.otf'),
-        'EuclidCircularB-MediumItalic': require('../assets/fonts/EuclidCircularB-MediumItalic.otf'),
+        'CentraNo1-Book': require('../assets/fonts/CentraNo1/CentraNo1-Book.otf'),
+        'CentraNo1-BookItalic': require('../assets/fonts/CentraNo1/CentraNo1-BookItalic.otf'),
+        'CentraNo1-Medium': require('../assets/fonts/CentraNo1/CentraNo1-Medium.otf'),
+        'CentraNo1-MediumItalic': require('../assets/fonts/CentraNo1/CentraNo1-MediumItalic.otf'),
+        'CentraNo1-Bold': require('../assets/fonts/CentraNo1/CentraNo1-Bold.otf'),
+        'CentraNo1-BoldItalic': require('../assets/fonts/CentraNo1/CentraNo1-BoldItalic.otf'),
```

Comment:
Replacing EuclidCircularB with CentraNo1
- New nested folder as we are likely to add MMSans and MMPoly support in a subsequent PR
- `Regular` is replaced by `Book`
- Only font files that exist are listed
![Screenshot 2025-03-21 at 2 30 20 PM](https://github.com/user-attachments/assets/ebc3a3b8-bb12-46bf-9cd3-80e8569001d9)